README.md

```diff
@@ -8,13 +8,22 @@ This project contains:
 - [lsp4jakarta](/lsp4jakarta) - Language Server for Jakarta EE
 - [jakarta-eclipse](/jakarta-eclipse) - Eclipse JDT LS extension and Eclipse IDE client that consumes the Language Server for Jakarta EE
 
+# Table of Contents  
+- [Getting Started](##Getting-Started)  
+    - [Prerequisites](###Prerequisites)
+    - [Setup Instructions](###Setup-Instructions)
+- [Eclipse Plugin Installation Instuctions](##Eclipse-Plugin-Installation-Instuctions)
+- [Common Errors](##Common-Errors)
+- [Contributing](##Contributing)
+- [Feedback](##Feedback)
```

Comment:
To link to other sections in the same document, only one `#` is needed. ie. `[Feedback](#Feedback)`

Reply:
Consider reorganizing the document as follows
```
# Table of Contents  
- [Getting Started](#Getting-Started)  
    -  in this section, put the new instructions on how to install the latest jar through the dropins directory
- [Building](#Building)
    -  in this section, we can link to a separate "Building" document (create under `docs/Building.md`). In this document we can put the Prequisites, Setup Instructions, and Common Errors sections.
- [Contributing](#Contributing)
- [Feedback](#Feedback)
```
What do you think?